app/views/adopters/_dog_choices.html.erb

```diff
@@ -1,7 +1,16 @@
 <table class="table table-striped table-bordered table-condensed">
   <tr>
     <th>Pet Type</th>
-    <td><%= @adopter.dog_or_cat%></td>
+    <% if @adopter.dog_or_cat? %>
+      <td><%= @adopter.dog_or_cat%></td>
+    <% else %>
+      <td>
+        <%= form_for @adopter, format: :json, html: { class: 'form-horizontal' } do |f| %>
+        <%= f.select :dog_or_cat, options_for_select([['Dog'], ['Cat']]) %>
```

Comment:
The drop down should default to a blank value, otherwise it shows as "Dog" being already selected even though it isn't assigned.

`{ include_blank: true }`  
https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select

Reply:
Thanks!